rpcs3/Emu/RSX/VK/VKFormats.cpp

```diff
@@ -39,6 +39,32 @@ VkFormat get_compatible_depth_surface_format(const gpu_formats_support &support,
 	throw EXCEPTION("Invalid format (0x%x)", format);
 }
 
+std::tuple<VkFilter, VkSamplerMipmapMode> get_min_filter_and_mip(u8 min_filter)
+{
+	switch (min_filter)
+	{
+	case CELL_GCM_TEXTURE_NEAREST: return std::make_tuple(VK_FILTER_NEAREST, VK_SAMPLER_MIPMAP_MODE_NEAREST);
```

Comment:
It's often "just in case". I often add or remove elements in function signature, adding an element to a 2-tuple is easier than turning a pair into a 3-tuple.